clinical-domain-agent/pom.xml

```diff
@@ -32,6 +32,24 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>io.swagger.core.v3</groupId>
+      <artifactId>swagger-core</artifactId>
+      <version>2.2.26</version>
+    </dependency>
+
+    <dependency>
+      <groupId>jakarta.xml.bind</groupId>
+      <artifactId>jakarta.xml.bind-api</artifactId>
```

Comment:
Spring boot dependencies already specifies `jakarta.xml.bind-api`s version, see [maven central](https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/3.4.0). Therefor we should be able to omit the explicit version here.

Reply:
No, we need this exact version. Without this swagger-core throws an exception. Looks like a bug in swagger-core and/or springdoc-openapi-starter-webflux-ui to me.